lms/djangoapps/discussion/django_comment_client/base/tests.py

```diff
@@ -264,7 +268,6 @@ def _setup_mock_request(self, mock_request, include_depth=False):
         Ensure that mock_request returns the data necessary to make views
         function correctly
         """
-        mock_request.return_value.status_code = 200
```

Comment:
There are still other tests that set this value explicitly. Should they be removed from those other tests as well?

Reply:
I will check this in more detail tomorrow and update.

Reply:
Those who set this value expicitly will set after the call of our code. so this is good to go.

Reply:
If they set it after the call of our code, it seems like those lines are no longer needed right? so we should remove them?

Reply:
done.